Communitheme/gtk-3.0/_common.scss

```diff
@@ -511,8 +517,8 @@ $_dot_color: if($variant=='light', $selected_bg_color,
 %button,
 button {
   @at-root %button_basic, & {
-animation: shrink 1s linear infinite;
-    min-height: 24px;
```

Comment:
I think the button `min-height` and `min-width` should hold to `24px`. Since some buttons have `24px` icons, using less than `24px` sometimes results in irregular button heights.

Instead we can reduce `padding`.